calendar.js

```diff
@@ -58,46 +105,74 @@ function createTable() {
 }
 //function which makes an ajax call to get the values from the json file which contains the hours for which appointments have been booked
 function getScheduleAndUpdateTable(table) {
-    var time = [];
+    var eventname=[];
+    var starttime = [];
+    var endtime=[];
+    var locationname=[];
     $.ajax({
-        url: './calendar.json',
+        url: '/getJSONResponse',
```

Comment:
nice url, but it's more conventional to use all smallcases for apis, if you like, update, else leave it. Your choice